packages/material-ui/src/OutlinedInput/OutlinedInput.test.js

```diff
@@ -28,4 +28,10 @@ describe('<OutlinedInput />', () => {
 
     expect(container.querySelector('.notched-outlined')).not.to.equal(null);
   });
+
+  it('should add the classes when size is small', () => {
+    const { container } = render(<OutlinedInput size="small" id="smInput" />);
```

Comment:
```suggestion
    const { container } = render(<OutlinedInput size="small" />);
```

If you need id I would recommend using `test-id` , but in this case it may not even be required.